tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -2129,6 +2131,97 @@ def legacy_fully_connected(x,
 
     return _apply_activation(y, activation_fn, output_collections)
 
+def adaptive_softmax_loss(inputs,
+                          labels,
+                          cutoff,
+                          project_factor=4,
+                          initializer=None,
+                          name=None):
+  """Computes and returns the adaptive softmax loss (a improvement of 
```

Comment:
The initial line should be a complete summary that ends in a period (not two lines like you have here)